packages/features/shell/Shell.tsx

```diff
@@ -172,23 +172,38 @@ const CustomBrandingContainer = () => {
   return <CustomBranding lightVal={user?.brandColor} darkVal={user?.darkBrandColor} />;
 };
 
-export default function Shell(props: LayoutProps) {
-  useRedirectToLoginIfUnauthenticated(props.isPublic);
-  useRedirectToOnboardingIfNeeded();
-  useTheme("light");
-  // don't load KBar when unauthed
-  return props.isPublic ? (
-    <>
-      <CustomBrandingContainer />
-      <Layout {...props} />
-    </>
-  ) : (
+const PublicShell = (props: LayoutProps) => {
+  const { status } = useSession();
```

Comment:
Could we do something like this instead?

Const wrapper = status === "authenticated" ? KBarRoot : Fragment

  <Wrapper>
      <CustomBrandingContainer />
      <Layout {...props} />
    </Wrapper>

Reply:
<Wrapper>
      <CustomBrandingContainer />
      <Layout {...props} />
  </Wrapper>